src/helpers/helpers.ts

```diff
@@ -355,12 +387,45 @@ export class Helpers {
       { name: 'resolutionModule', address: request.resolutionModule },
       { name: 'finalityModule', address: request.finalityModule },
     ];
+
     requestModules.forEach((requestModule) => {
       if (
         !(requestModule.address == '0x0000000000000000000000000000000000000000' || !requestModule.address) &&
-        !Object.keys(this.modules.knownModules).includes(requestModule.address as string)
+        !Object.keys(knownModules).includes(requestModule.address as string)
       )
         throw new Error(`${requestModule.name}: ${requestModule.address} is not a known module`);
     });
   }
+
+  private mapEventArgsToRequestWithId(event: any[]): RequestWithId {
+    return {
+      requestId: event[0],
+      request: event[1],
+      ipfsHash: event[2],
+      blockNumber: event[3],
+    };
+  }
+
+  private mapEventArgsToResponseWithId(event: any[]): ResponseWithId {
+    return {
+      requestId: event[0],
+      responseId: event[1],
+      response: event[2],
+      blockNumber: event[3],
+    };
+  }
+
+  private mapEventArgsToDisputeWithId(event: any[]): DisputeWithId {
+    return {
+      responseId: event[0],
+      disputeId: event[1],
+      dispute: event[2],
+      blockNumber: event[3],
+    };
+  }
 }
+
+export const getDecodeRequestDataFunctionReturnTypes = (abi: any[]) => {
+  const decodeRequestDataFunction = abi.find((item) => item.name === 'decodeRequestData').outputs;
+  return decodeRequestDataFunction;
+};
```

Comment:
What's the difference between this function and `getDecodeRequestReturnTypes` in module.ts?